src/patternfly/components/DescriptionList/description-list.scss

```diff
@@ -82,6 +82,7 @@ $pf-c-description-list--breakpoint-map: build-breakpoint-map("base", "sm", "md",
 .pf-c-description-list__term,
 .pf-c-description-list__description {
   text-align: left;
+  overflow-wrap: break-word;
```

Comment:
This still results in this kind of overflow. If that's what you're trying to avoid, we use `word-break: break-word;` most places.

<img width="627" alt="Screen Shot 2021-03-10 at 5 56 41 PM" src="https://user-images.githubusercontent.com/35148959/110715006-090a9f00-81ca-11eb-8046-4bf4f2f1a3fd.png">

Reply:
Fixed this